<launch>

  <node pkg="nodelet" type="nodelet" name="realsense_camera_manager" args="manager" output="screen"/>

  <node pkg="nodelet" type="nodelet" name="multi_realsense_camera" args="load realsense2_camera/MultiT265Odom  realsense_camera_manager">
    <param name="frame_prefix" value="realsense" />
    <param name="reset_duration" value="0.05" />
  </node>
</launch>
